resources/prefabs/bee.ron

```diff
@@ -0,0 +1,35 @@
+#![enable(implicit_some)]
```

Comment:
I think it would be great if you can split the bee into a separate pull request so that we can refer people to it if they ask how to add creatures.